docs/Python-Intro.rst

```diff
@@ -112,11 +112,11 @@ or
 
 And you can use ``Dataset.set_init_score()`` to set initial score, and ``Dataset.set_group()`` to set group/query data for ranking tasks.
 
-**Memory efficent usage:**
+**Memory efficient usage:**
 
 The ``Dataset`` object in LightGBM is very memory-efficient, due to it only need to save discrete bins.
 However, Numpy/Array/Pandas object is memory cost.
-If you concern about your memory consumption, you can save memory according to following:
+If your concern about your memory consumption, you can save memory according to the following:
```

Comment:
I think that `concern` is a verb here, so `you` is OK. Or you can change `concern` to `worry`.

Reply:
You are right, I updated it